boot/grub_menu.py

```
@@ -46,6 +47,21 @@
 #: מצבי סבב שבהם יש למה להעלות את הסוכן.
 JOINABLE_SESSION_STATES = frozenset({"open", "running"})
 
+#: התפקידים שערך ה-ImageCtl היחיד שלהם הוא הסוכן במצב **רגיל** (#144).
+#: כל תפקיד אחר מקבל את הערך במצב `recovery`, וזה לא שרירותי — זו טבלת
+#: ההחלטה של הסוכן עצמו (`agent/lib/decide.sh`) קרואה מהצד השני:
+#:
+#:   build      → `build_console`  — מסך הקליטה/ההפצה, הדרך היחידה אליו
+#:                מאתחול קר (#29). ‏recovery היה מסתיר אותו.
+#:   classroom  → `local`, כלומר `die_local` ו-`reboot -f`. ערך "רגיל"
```

Ruff (0.16.2)

[error] Contains control characters that can permit obfuscated code

(PLE2502)